repo/packages/L/linkerd/10/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "linkerd",
+  "version": "1.0.0-0.1",
+  "scm": "https://github.com/linkerd/linkerd/",
+  "description": "A dynamic linker for microservices. Install, set your http proxy to $HOST:4140, and connect to your services via app name. linkerd will handle Marathon service discovery for you.",
+  "maintainer": "linkerd-users@googlegroups.com",
```

Comment:
@ryadav88 `linkerd-users@googlegroups.com` is our primary mailing list, is that ok?